airrohr-firmware/airrohr-firmware.ino

```diff
@@ -2257,7 +2257,6 @@ void wifiConfig() {
 		const IPAddress apIP(192, 168, 4, 1);
```

Comment:
This is needed for users that forgot the password for the sensor in AP mode.
And we need this for a future function where we create a unique password at first start. So this will be the only possible way to read this password.
You need physical access to read this (debug output over USB only). So this should not be too sensitive. It's not the password for the wifi the sensor will connect to.

Reply:
thanks for the explanation. I've updated this commit to log this with a prefix again.